lib/dropdown/index.js

```diff
@@ -0,0 +1,46 @@
+import angular  from 'angular';
+import box      from '../box/index';
+import icons    from '../icons/index';
+
+import DropdownCtrl from './DropdownCtrl';
+
+var dropdown = angular.module('guDropdown', ['guBox', 'guIcons', 'guDropdownCtrl']);
+
+dropdown.directive('guDropdown', function dropdownDirective(){
+  return {
+    transclude: true,
+    template: '<div class={{className}} ng-transclude></div>',
+    restrict: 'E',
+    scope: { state: '@' },
+    controller:'guDropdownCtrl'
```

Comment:
Ive tried playing with this but can't make it work. The controller is here to be passed to `guDropdownTrigger` so It can be called on click. Maybe I'm missing something?